README_FRAME_FORMAT.rst

```diff
@@ -62,7 +68,9 @@ Then it follows the info about the filter pipeline.  There is place for a pipeli
       |   +--number of filters
       +--[msgpack] fixext 16
 
-In addition, a frame can be completed with meta-information about the stored data; these data blocks are called metalayers and it is up to the user to store whatever data they want there, with the only (strong) suggestion that they have to be in the msgpack format.  Here it is the format for the case that there exist some metalayers::
+At the end of the frame, meta-information about the chunked data can be stored in data blocks are called metalayers.
```

Comment:
I suggest the next rephrasing:

At the end of the frame come the so-called metalayers, that is, meta-information about the data stored in the frame.